cs3client/file.py

```diff
@@ -170,7 +170,8 @@ def touch_file(self, auth_token: tuple, resource: Resource) -> None:
         self._log.debug(f'msg="Invoked TouchFile" trace="{res.status.trace}"')
 
     def write_file(
-            self, auth_token: tuple, resource: Resource, content: Union[str, bytes], size: int, lock_md: tuple = None
+            self, auth_token: tuple, resource: Resource, content: Union[str, bytes], size: int,
+            app_name: str = None, lock_id: str = None
```

Comment:
Shouldn't it be `Union[str, None]`?

Reply:
Right, or even better `Optional[str] = None`. Will fix all cases.